opm/autodiff/MultisegmentWell.hpp

```diff
@@ -115,13 +115,14 @@ namespace Opm
         virtual void assembleWellEq(const Simulator& ebosSimulator,
                                     const std::vector<Scalar>& B_avg,
                                     const double dt,
+                                    const double strictBHP,
                                     WellState& well_state,
```

Comment:
Passing strictBHP like this is quite ugly. Any idea how to improve this?

Reply:
We will discuss the interface things later.